pubspec.yaml

```diff
@@ -0,0 +1,123 @@
+name: mega_trust_project
```

Comment:
Always choose meaningful names for your project name since it will appear in imports.